test/__snapshots__/loader.test.js.snap

```diff
@@ -123,18 +72,15 @@ exports[`Loader styled 0: styled-FancyBox.css 1`] = `
   width: 75px;
 
   @media (min-width: 420px) {
-		width: 96px;
-		height: 96px;
-	}
-
-  
-
-}
-.fancy-box-variant-0 {
+    width: 96px;
+    height: 96px;
+  }
 
+  &.primary {
```

Comment:
this essentially requires that you have a nesting api set up, which means no usage with plain css..